source/core/ui/view/map/OpenLayerView.js

```diff
@@ -165,36 +162,33 @@ class OpenLayerView extends MapView {
     initMap(options) {
 
         this.map = null;
-        let initialView = null;
         this.first = true;
         let overlays = [];
 
         let baseLayers = this.getDefaultLayers();
         let maxZoom = 19;
 
+        let view = new OlView({
```

Comment:
If options.projection were to be available could we not set it here in case we do not want a default projection?  I ask only  to try an future proof ourselves.

Reply:
As per conversation with mdhsl, this is a non-issue